File: testprojects/c#/domain/language/babbel/LearnArabic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.asocialmedia;

namespace CSharpBenchmark.domain.language.babbel
{
    //Functional requirement 3.1.3
    //Test case 91: Class domain.language.babbel.LearnArabic must use infrastructure.socialmedia.SocialNetwork 
    //Result: FALSE
    public class LearnArabic
    {
        public LearnArabic()
        {
            //FR5.1
            new ASocialNetwork();
        }
    }
}

File: testresources/java/benchmark_2012/domain/language/busuu/LearnArabic.java
package domain.language.busuu;

import infrastructure.socialmedia.SocialNetwork;

//Functional requirement 3.1.3
//Test case 90: Class domain.language.busuu.LearnArabic must use class infrastructure.socialmedia.SocialNetwork
//Result: TRUE
public class LearnArabic {	
	public LearnArabic(){
		//FR5.1
		new SocialNetwork();
	}
}